superset/common/query_context_factory.py

```diff
@@ -54,6 +54,7 @@ def create(
         result_format: ChartDataResultFormat | None = None,
         force: bool = False,
         custom_cache_timeout: int | None = None,
+        viz_type: Optional[str] = None # CCCS -- This field is added to the query_context column in the slice table in our database. Adding this optional argument so that it does not cause an error
```

Comment:
Is this the mysterious property whose reasons for existence we're unsure of? If so, can we make a ticket to try to remove this in future (if we think we might not need it anymore)?

Reply:
Can we? I feel like there are potential dashboards with that property in the database and we need to deal with it because that property is buried in a json.

Reply:
Yeah, I have no idea how deep this can of worms goes. If we don't use this anymore though, I'm tempted to try to fix those dashboards. If someone tells me the dragons are too big though, nbd.